Microsoft.ML.sln

```diff
@@ -274,6 +274,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.ML.FastTree", "Mi
 		pkg\Microsoft.ML.FastTree\Microsoft.ML.FastTree.symbols.nupkgproj = pkg\Microsoft.ML.FastTree\Microsoft.ML.FastTree.symbols.nupkgproj
 	EndProjectSection
 EndProject
+Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Microsoft.ML.FSharp.Samples", "docs\samples\Microsoft.ML.Samples.FSharp\Microsoft.ML.FSharp.Samples.fsproj", "{370CF53B-2C46-49CF-86AA-D9CC77940E9D}"
```

Comment:
>FSharp.Samples [](start = 66, length = 14)

why is the proj "FSharp.Samples" and the folder name "Samples.FSharp"? let's keep them consistent